third_party/googletest/src/googlemock/include/gmock/gmock-nice-strict.h

```diff
@@ -225,8 +225,11 @@ class GTEST_INTERNAL_EMPTY_BASE_CLASS StrictMock
       "https://google.github.io/googletest/"
       "gmock_cook_book.html#NiceStrictNaggy");
   StrictMock() : MockClass() {
+#if !GTEST_OS_STARBOARD
     static_assert(sizeof(*this) == sizeof(MockClass),
                   "The impl subclass shouldn't introduce any padding");
+#endif // !GTEST_OS_STARBOARD - gives MSVC comp error, i think due to Starboard
```

Comment:
I'll add a few nitpicks: This should have at least a follow-up bug logged and in the comment here.

Reply:
Added bug b/288166017